Commands/gemini.py

```
@@ -72,3 +72,25 @@ def generate(prompt) -> list[str]:
+        model = GenerativeModel(
+            "gemini-1.5-flash-002",
+            system_instruction=system_instruction
+        )
+        response = model.generate_content(
+            prompt,
+            generation_config=generation_config,
+            safety_settings=safety_settings,
+            stream=False,
+        ).text.replace('\n', ' ')
+        response = response.replace('*', ' ')
+        n = 495
```

Ruff

Local variable `n` is assigned to but never used

Remove assignment to unused variable `n`

(F841)